backends/deeploy/pact_export.py

```diff
@@ -245,6 +256,10 @@ def export_net(net: nn.Module,
         enable_skip_layer_norm=False,
         enable_bias_gelu=False,
     )
+
+    if onnxruntime.__version__ >= "1.17":
```

Comment:
This looks pretty wrong; you are comparing strings here, not numbers. Please check this thread for info on doing this properly:
https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

Reply:
Addressed in [this commit](https://github.com/pulp-platform/quantlib/pull/15/commits/abfb6b0448b61ed8d979a4bf29f5124fab3f2d63)